core/client/src/main/java/alluxio/client/file/options/SetAttributeOptions.java

```diff
@@ -233,7 +254,11 @@ public SetAttributeTOptions toThrift() {
     }
     if (mTtl != null) {
       options.setTtl(mTtl);
+      options.setTtlExpiryAction(mTtlExpiryAction == TtlExpiryAction.FREE
```

Comment:
or better, the TtlExpiryAction should provide toThrift() and fromThrift() utility methods